src/main/java/mate.academy.internetshop/Factory.java

```diff
@@ -0,0 +1,53 @@
+package mate.academy.internetshop;
+
+import mate.academy.internetshop.dao.BucketDao;
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.dao.OrderDao;
+import mate.academy.internetshop.dao.UserDao;
+import mate.academy.internetshop.dao.impl.BucketDaoImpl;
+import mate.academy.internetshop.dao.impl.ItemDaoImpl;
+import mate.academy.internetshop.dao.impl.OrderDaoImpl;
+import mate.academy.internetshop.dao.impl.UserDaoImpl;
+import mate.academy.internetshop.service.BucketService;
+import mate.academy.internetshop.service.ItemService;
+import mate.academy.internetshop.service.OrderService;
+import mate.academy.internetshop.service.UserService;
+import mate.academy.internetshop.service.impl.BucketServiceImpl;
+import mate.academy.internetshop.service.impl.ItemServiceImpl;
+import mate.academy.internetshop.service.impl.OrderServiceImpl;
+import mate.academy.internetshop.service.impl.UserServiceImpl;
+
+public class Factory {
+
+    public static ItemDao getItemDao() {
+        return new ItemDaoImpl();
```

Comment:
@RomanMinevich тут краще було б використати Singleton
```suggestion
        if (itemDao == null) {
            itemDao = new ItemDaoImpl();
        }
        return itemDao;
```